lib/src/auto_size_text_field.dart

```diff
@@ -239,8 +239,7 @@ class AutoSizeTextField extends StatefulWidget {
   ///  * [AdaptiveTextSelectionToolbar], which is built by default.
   final EditableTextContextMenuBuilder? contextMenuBuilder;
 
-  static Widget _defaultContextMenuBuilder(
-      BuildContext context, EditableTextState editableTextState) {
+  static Widget _defaultContextMenuBuilder(BuildContext context, EditableTextState editableTextState) {
```

Comment:
Wondering if this linting and auto fix on the lengths of lines is a personal style only in your IDE or the default style from [Dart Lint ](https://pub.dev/packages/lint)?

Reply:
my personal style, you can ignore it